Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCocoa.mm

```diff
@@ -128,18 +128,18 @@
     , m_resourceBaseURL(resourceBaseURL)
     , m_manifestJSON(JSON::Value::null())
 {
-    RELEASE_ASSERT(m_bundle || m_resourceBaseURL);
+    RELEASE_ASSERT(m_bundle || !m_resourceBaseURL.isEmpty());
 
     auto *bundleResourceURL = m_bundle.get().resourceURL.URLByStandardizingPath.absoluteURL;
-    if (!m_resourceBaseURL && bundleResourceURL)
+    if (!m_resourceBaseURL.isEmpty() && bundleResourceURL)
         m_resourceBaseURL = bundleResourceURL;
 
 #if PLATFORM(MAC)
     m_shouldValidateResourceData = m_bundle && [m_resourceBaseURL isEqual:bundleResourceURL];
 #endif
 
-    RELEASE_ASSERT(m_resourceBaseURL.get().isFileURL);
-    RELEASE_ASSERT(m_resourceBaseURL.get().hasDirectoryPath);
+    RELEASE_ASSERT(m_resourceBaseURL.protocolIsFile());
+    RELEASE_ASSERT(m_resourceBaseURL.hasPath());
```

Comment:
This likely does not translate, since the previous assert was basically looking for a trailing `/` on the path.

Reply:
I can add another check to see if the last character is a '/'